client/state/ui/first-view/selectors.js

```diff
@@ -78,14 +90,13 @@ function routeSetIsInCurrentSection( state, routeSet ) {
 	return some( section.paths, path => startsWith( routeSet.path, path ) );
 }
 
+// TODO: memoize
```

Comment:
are leaving this TODO in so it's not forgotten in future or will you address it now?